intermediate/indexing/advanced-indexing.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
```

Comment:
<p>typo in Exercises.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/264/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>extra space before the colon (before {solution})</p>

Reply:
Should be fixed now! Thanks for noticing these.